.claude/commands/pr-report.md

```
@@ -0,0 +1,117 @@
+   - Extract key information from CodeRabbit summary or PR description
+
+4. **Categorize by module**
+   - Analyze PR titles and descriptions
+   - Group PRs by major modules (e.g., Market, WebDapp, Swap, Discovery, etc.)
+   - Use "通用组件" for general/shared components
+
+5. **Generate Slack-friendly report**
+   - Use plain text format (no Markdown links)
+   - Use `———` for section dividers
+   - Format:
+     ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)